OpenRA.Mods.Common/Lint/CheckPlayers.cs

```diff
@@ -89,6 +89,20 @@ public void Run(Action<string> emitError, Action<string> emitWarning, ModData mo
 						emitError("Actor {0} needs to be owned by the player that owns the world. ".F(kv.Key) +
 							"Use the `Spawn` and `LockSpawn` player properties to force players onto a particular spawn instead.");
 					}
+
+					// Some actors may have RequiresSpecificOwner to enforce a specific owner (example: 'Neutral' for trees)
+					foreach (var actor in map.Rules.Actors)
```

Comment:
Why are we enumerating all actors? We already know exactly what actor we want to look up here!